integration/src/main/java/scripts/testscripts/GetStatus.java

```diff
@@ -0,0 +1,31 @@
+package scripts.testscripts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.axonserver.api.PublicApi;
+import bio.terra.axonserver.client.ApiClient;
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import com.google.api.client.http.HttpStatusCodes;
+import scripts.utils.ClientTestUtils;
+
+public class GetStatus extends TestScript {
```

Comment:
how is this test run? 

I'm pretty sure you need to set up config first. let's remove the integraiton test for now. the unit test is sufficient.

Reply:
Removing integration test